lib/mesa-lib/DEPENDS

```diff
@@ -14,6 +14,7 @@ depends presentproto
 depends Mako
 depends nettle
 
+optional_depends wayland  "" "" "This is needed by various components of Plasma"
```

Comment:
I believe  --with-egl-platforms=wayland is a requirement as well. Since we add x11 and drm manually I doubt auto-detection really kicks in.